src/v/coproc/offset_utils.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "bytes/bytes.h"
+#include "coproc/ntp_context.h"
+#include "model/fundamental.h"
+#include "reflection/adl.h"
+
+namespace coproc {
+
+using map_type = ntp_context::offsets_map;
```

Comment:
i think this obscures the type imo. `ntp_context::offset_map` sounds better to me.